_utility/deploy_prb.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -e
+
+if [ "$#" -ne 3 ]; then
+  echo "Invalid argument count."
+  echo "Usage: $0 AWS_REGION WEBSITE_INDEX_DOCUMENT BUCKET_NAME"
+  exit 1
+fi
+
+cd _site/
+
+AWS_REGION=$1
+WEBSITE_INDEX_DOCUMENT=$2
+BUCKET_NAME=$3
+
+bucketDoesNotExist() {
+  number_of_buckets_with_name=$(aws s3 ls | awk '{print $3}' | grep -c "^$1$") 
+  echo $(expr $number_of_buckets_with_name = "1") == 0 
+}
+
+BUCKET_NOT_EXISTS=$(bucketDoesNotExist $BUCKET_NAME)
+if [ $BUCKET_NOT_EXISTS ]; then
+  aws s3 mb s3://$BUCKET_NAME --region $AWS_REGION
+  aws s3 website s3://$BUCKET_NAME --index-document $WEBSITE_INDEX_DOCUMENT	
+fi
+
+aws s3 sync . s3://$BUCKET_NAME --acl public-read
+aws cloudfront create-invalidation --distribution-id EML2BUMD54HSC --paths "/*"
```

Comment:
We need to find out if this distribution id `EML2BUMD54HSC` can be reused for our jobs, or do we need a new one, maybe @mashooq can recommend us something